### PR TITLE
fixes #11195 - handle foreman restructuring of kickstart_attributes

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: app/lib/katello/concerns/renderer_extensions.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        alias_method_chain :kickstart_attributes, :katello
+      end
+
+      def kickstart_attributes_with_katello
+        kickstart_attributes_without_katello
+
+        if @host.content_view && @host.operatingsystem.is_a?(Redhat) &&
+          !@host.operatingsystem.kickstart_repo(@host).blank?
+
+          @mediapath = @host.operatingsystem.mediumpath(@host)
```
Comment (jlsherrill) [inline app/lib/katello/concerns/renderer_extensions.rb]: I might change this to @mediapath ||=

just in case the user overrode it

Comment (jlsherrill) [inline app/lib/katello/concerns/renderer_extensions.rb]: actually this doesn't actually matter as the mediumpath code will actually use the install media if its there

Comment (jlsherrill) [inline app/lib/katello/concerns/renderer_extensions.rb]: well it kind of does.  If the user explicitly sets an install media url, we don't want to override that

Comment (stbenjam) [inline app/lib/katello/concerns/renderer_extensions.rb]: @jlsherrill Makes sense, updated!

**Comment by jlsherrill** [comment]
APJ

**Pull request merged**